GUI/Start-SysAdminSuiteGui.ps1

```
@@ -2326,6 +2510,239 @@ $btnOpenMIOutput.Add_Click({
+$picQRPreview.BorderStyle = 'FixedSingle'
+$picQRPreview.BackColor = [System.Drawing.Color]::White
+$picQRPreview.Visible = $false
+
+$lblQRPreviewHint = New-Object System.Windows.Forms.Label
+$lblQRPreviewHint.Location = '330,412'
+$lblQRPreviewHint.Size = '420,48'
+$lblQRPreviewHint.Anchor = 'Top, Left, Right'
+$lblQRPreviewHint.Font = $uiFont
+$lblQRPreviewHint.Text = 'Review the payload preview, then click Generate QR. QR generated locally. Scan to paste selected payload.'
+
+function Update-QRGeneratorPreview {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Update-QRGeneratorPreview' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)